example/example_SG.md

```diff
@@ -0,0 +1,101 @@
+# Example Style Guide English, USA (en-US)
+
+This style guide provides instruction for translating to the American English locale. This style guide is to be used in coordination with the [General Mozilla L10n Style Guide](general_L10n_SG.md)
+
+## Formality and tone
+
+In English, the tone of Mozilla's language should be adaptable to anyone. From new users to season Mozillians, the tone should be approachable, welcoming, and inclusive.
+
+Mozilla is fun and should delight the user in expected and unexpected ways. The level of formality should express a sense of familiarity and friendliness while maintaining a voice of authenticity.
+
+In general, use the second person (`you`) to address the user in Firefox products. Firefox is a user agent, and this point of view reinforces the message that "Firefox works for you."
+
+Avoid using the first person (`my`, `I`) as it can create confusion about who is being addressed.
+
+In some projects (like Desktop, Android, IOS, Focus), the language will be direct as it belongs to a software string. This language should be maintained as direct, yet respectful, and remain casual as with all other projects. Casual language should never be used at the expense of sounding disrespectful, regardless of the project.
+
+## Natural expression
+
+Translations should sound as natural as possible. Borrowed words should never be used (with the exception of branded names).
+
+## Cultural references, idioms, and slang
+
+When dealing with cultural references, idioms, and slang in English, references to American pop culture (music, movies), sports (basketball, baseball, football), sayings, and some proverbs will be generally recognized. If a cultural reference is difficult to express in English, use the meaning to find a good equivalent. Refer to sources and databases of common expressions to help find an adequate equivalent.
```

Comment:
If this is a style guide for en-US, as the beginning claims, do we really need this paragraph?

Reply:
I believe so because we want this style guide (which should never actually be used) to serve as a comprehensive example for communities as they draft their own. For this style guide to work as an example, it is drafted as though the source language is not identical to the style guide language, however we remain conscious of the linguistic features of the source language (which in reality is identical). 

I'm happy to discuss other options too :)